doc/syntax.jax

```diff
@@ -2170,6 +2242,16 @@ Makefileではエラーを見つけやすくするためにコマンドがハイ
 
 	:let make_no_commands = 1
 
+コメントもデフォルトでハイライトされる。これをオフにするには: >
+
+	:let make_no_comments = 1
+
+Microsoft Makefile は変数の展開とコメントを異なる方法で処理する (エスケープに
+バックスラッシュは使用されない)。このため間違ったハイライトが表示される場合は、
+以下を試すこと: >
```

Comment:
```suggestion
バックスラッシュは使用されない)。このために間違ったハイライトが表示される場合
は、以下を試すこと: >
```

感覚の話になりますが、「このため間違った〜」は、`~. Therefore ~` や `~. So ~` のような文章の訳っぽくて、`because of this` の訳っぽくないように感じたので、提案です。